src/screen.rs

```diff
@@ -609,14 +609,19 @@ impl Screen {
 
         // read new data
         let mut data = vec![];
+
+        #[allow(unused_must_use)]
```

Comment:
I'd feel it be better this be removed and instead mapping the Result fail to a `{}` or something